ui/packages/consul-ui/docs/bookmarklets.mdx

```diff
@@ -12,7 +12,7 @@ Below is a list of the most commonly used functions as bookmarklets followed by
 | [Enable ACLs](javascript:Scenario('CONSUL_ACLS_ENABLE=1')) | Enable ACLs |
 | [Enable TProxy](javascript:Scenario('CONSUL_TPROXY_ENABLE=1')) | Enable TProxy |
 | [Enable Nspaces](javascript:Scenario('CONSUL_NSPACES_ENABLE=1')) | Enable Namespace Support |
-| [Enable Peers](javascript:Scenario('CONSUL_PEERING_ENABLE=1')) | Enable Peers Support |
+| [Enable Peers](javascript:Scenario('CONSUL_PEERINGS_ENABLE=1')) | Enable Peering Support |
```

Comment:
do we need to update `get-environment` as well? I think this is where this cookie value is being read. And we are changing from `CONSUL_PEERING_ENABLE` to `CONSUL_PEERINGS_ENABLE` here.

Reply:
I missed a couple 😬 I've added those and searched the codebase for references to that to triple check, thanks for the nudge!